src/main/java/org/jabref/gui/fieldeditors/identifier/IdentifierEditor.java

```diff
@@ -90,6 +99,15 @@ public IdentifierEditor(Field field,
         new EditorValidator(preferences).configureValidation(viewModel.getFieldValidator().getValidationStatus(), textArea);
     }
 
+    private void initKeyBindings() {
+        addEventFilter(KeyEvent.KEY_PRESSED, event -> {
+            Optional<KeyBinding> keyBinding = preferences.getKeyBindingRepository().mapToKeyBinding(event);
+            if (keyBinding.isPresent() && keyBinding.get() == KeyBinding.GET_BIBLIOGRAPHIC_DATA) {
+                this.fetchInformationByIdentifier();
+            }
+        });
```

Comment:
@koppor Just because I couldnt get it to work with org.jabref.gui.actions.ActionFactory#configureIconButton (locally).
And debugging it, didnt help either, i tried to understand it by using org.jabref.gui.groups.GroupDialogView#GroupDialogView (it configures a HelpAction) as a reference. 
But also this doesnt seem to work for me locally. 

When i have following modal open:
![grafik](https://github.com/user-attachments/assets/a5034cd3-cb0e-456b-83d2-becdbf9e60fc)

and press F1, nothing happens.

Then i stumpled upon org.jabref.gui.frame.JabRefFrame#initKeyBindings
which was the only occurence where I could observe a keybinding to be actually caught.

And then I copied that code and at least i now i have the wanted behavior.

Reply:
It sounds like the keyevent is consumed before reaching the Dialog... Maybe you can try to follow the keyevent?

Reply:
The event doesnt Seem to be  Consumed.
I also debugged a bit further, it seems like this issue affects all Buttons that are in some sort of a dialog.
Buttons that did not work:
autosaveLocalLibrariesHelp in org.jabref.gui.preferences.general.GeneralTab
remoteHelp in org.jabref.gui.preferences.general.GeneralTab

But there are also buttons that work that way, like:
generateCitationKeyButton in org.jabref.gui.fieldeditors.CitationKeyEditor

I Suspect that it matters where The Button is mounted. 

But this for sure seems like a bug to me, could some try to reproduced it to verify?

Reply:
I have a mac and there the KeyEvent is always going through the top menu, so F1 always triggers the "online help" menu item and thus opens the default help page.
So I cannot help here

Reply:
The only hint I currently have is that one should try with a minimal JavaFX app. Maybe https://github.com/Siedlerchr/javafxreproducer?